SDK-1528: Add profile sandbox example (development > master) (#8)
getyoti/yoti-node-sdk-sandbox: README.md

* `/path/to/your-pem-file.pem` is the path to the application pem file. It can be downloaded only once from the Keys tab in your Yoti Hub.

* Point the Yoti client at the sandbox by setting environment variable `YOTI_CONNECT_API` to https://api.yoti.com/sandbox/v1

> Please do not open the pem file as this might corrupt the key and you will need to create a new application.

Reviewer: Don't need to create a new application any more:
```suggestion
Please do not open the pem file as this might corrupt the key and you will need to recreate the keys on the Yoti Hub.
```

Author: I'll open a separate PR for this as development is protected (this is just merging approved changes into master without release)
